caiman/base/movies.py

```diff
@@ -206,6 +206,86 @@ def motion_correct(self,
 
         return self, shifts, xcorrs, template
 
+    def motion_correct_3d(self,
```

Comment:
The version in `movies.py` for the motion correction is deprecated. It's still used for rigid motion correction during online processing sometimes but I wouldn't worry about that.